src/js/Text/index.ts

```diff
@@ -0,0 +1,56 @@
+import Control, {ControlProps} from '../Control';
+
+type TextProps = ControlProps & {
+  value?: string;
+}
+
+class Text extends Control {
+  protected _props: TextProps = {
+    ...this._props,
+    ...{
+      value: ''
+    }
+  }
+  private inputEl: HTMLInputElement;
+
+  constructor(params: TextProps) {
+    super();
+    if(typeof params !== 'undefined' && typeof params.isDisabled !== 'boolean') {
```

Comment:
if(typeof params !== 'undefined' && typeof params.isDisabled !== 'boolean') will throw error 'null is not an object' if null was specified, so we need to change the logic.
You can try check if(typeof params === 'object' && typeof params.isDisabled !== 'boolean')
But this still fails in case params is null because typeof null returns 'object'

Please, consider refactoring the check as following:
if(params === 'object' && params !== null && typeof params.isDisabled !== 'boolean')